build_parameters.json

```diff
@@ -184,16 +186,30 @@
 							"DESTINATION":     "step/controller:${BRANCH}-${ENVIRONMENT}-java-17",
 							"DESTINATION_PRD": "step/controller:${VERSION}-java-17",
 							"CONFIG": "controller.dockerfile",
-							"IMAGE_BASENAME": "step-controller-${VERSION}-java-17",
+							"IMAGE_BASENAME": "step-controller-${BRANCH}-${ENVIRONMENT}-java-17",
+							"RAW_DIRECTORY": "/step/${SIMPLIFIED_BRANCH_NAME}/sbom/",
 							"SHARE_ON_FTP": false,
 							"DOCKER_BUILD_ARGS": "--build-arg BASE_IMAGE=debian:12-slim --target java-17"
 						},
 						{
+							"DEACTIVATED": "PRODUCTION",
+							"ORIGIN": "step-distribution-controller/target/step-controller-${VERSION}.zip",
+							"DESTINATION":     "step/controller:${BRANCH}-${ENVIRONMENT}-java-21",
+							"DESTINATION_PRD": "step/controller:${VERSION}-java-21",
```

Comment:
If I understand the logic correctly, DESTINATION_PRD would be unused here because this snipped is deactivated for PRODUCTION. So I assume it is superfluous and can be removed?